app/models/user.rb

```diff
@@ -40,8 +40,11 @@ def last_lesson_completed
   end
 
   def image(size = 25)
-    #Could use GravatarUrlBuilder and users_helper
-    self.avatar ? self.avatar : "http://www.gravatar.com/avatar/436053b3e050d4156773bc04cfb167fe?s=#{size}"
+    if avatar
```

Comment:
How about

```ruby

def image(size = 25)
  avatar || odin_avatar(size)
end

private

def odin_avatar(size)
  "http://www.gravatar.com/avatar/436053b3e050d4156773bc04cfb167fe?s=#{size}"
end
```

Also, we might have to create a new odin gravatar to match with our new odin logo. What do you think?